Redis/legos/redis_get_keys_count/redis_get_keys_count.py

```diff
@@ -0,0 +1,32 @@
+#
+# Copyright (c) 2022 unSkript.com
+# All rights reserved.
+#
+
+import pprint
+from beartype import beartype
+from pydantic import BaseModel, Field
+
+class InputSchema(BaseModel):
+    pattern: str = Field(
+        title='Pattern',
+        description='Pattern for the searched keys')
+
+@beartype
+def redis_get_keys_count_printer(output):
+    if output is None:
+        return
+    pprint.pprint({"Matched keys count": output})
+
+@beartype
+def redis_get_keys_count(handle, pattern: str):
+    """redis_get_keys_count returns the matched keys count.
+
```

Comment:
Please add :param: pattern to the comment section as we use Sphinx to generate lego documentation.

Reply:
My fault, added. Thanks @jayasimha-raghavan-unskript